Installers/Node/index.js

```diff
@@ -99,8 +101,18 @@ function install() {
                     console.log("Injecting index.js");
 
                     var data = fs.readFileSync(_discordPath + _index).toString().split("\n");
-                    data.splice(_importSplice, 0, 'var _betterDiscord = require(\'betterdiscord\');\n');
-                    data.splice(_functionCallSplice, 0, splice);
+                    
+                    if(_os == "linux") {
```

Comment:
Actually, we can just remove the if clause here, #336 has it by default. Should be functional across the board.